adding new docker run instructions (#34)
Illumina/NirvanaDocumentation: docs/introduction/getting-started.md

## Getting Nirvana

### Latest Release

> Contact the team to obtain the latest release and or Docker image.

Reviewer: Should we add the email address you talked about today?

Author: yes, forgot to push changes

### Docker

> Assuming you have the docker image in a zip file (e.g. Nirvana-v3.21.0-net6.0-docker.tar.gz), you may load it as follows

Reviewer: ```suggestion
Obtain the docker image in a zip file (e.g. Nirvana-v3.21.0-net6.0-docker.tar.gz), and load it as follows
```

```bash
docker load < Nirvana-v3.21.0-net6.0-docker.tar.gz
```

For Docker, we have special instructions for running the Downloader:

```bash
docker run --rm -it -v local/data/folder:/scratch nirvana:v3.21.0 Downloader --ga GRCh37 -o /scratch
```

Similarly, we have special instructions for running Nirvana (Here's [a toy VCF](https://illumina.github.io/NirvanaDocumentation/files/HiSeq.10000.vcf.gz) in case you need it):

```bash
docker run --rm -it -v local/data/folder:/scratch nirvana:v3.21.0 Nirvana -c /scratch/Cache/ \
     -r /scratch/References/Homo_sapiens.GRCh37.Nirvana.dat \
     --sd /scratch/SupplementaryAnnotation/GRCh37 \
     -i /scratch/HiSeq.10000.vcf.gz -o /scratch/HiSeq
```
:::caution
We think Docker is fantastic. However, because our data files are usually accessed through a Docker volume, there is a noticeable performance penalty when running Nirvana in Docker.
:::
:::tip

> For convenience, the user is incouraged to create aliases for the docker commands. For example:

Reviewer: ```suggestion
For convenience, the user is encouraged to create aliases for the docker commands. For example:
```
